includes/widgets/video.php

```diff
@@ -1262,9 +1262,79 @@ private function render_hosted_video() {
 		}
 
 		$video_params = $this->get_hosted_params();
-		/* Sometimes the video url is base64, therefore we use `esc_attr` in `src`. */
-		?>
-		<video class="elementor-video" src="<?php echo esc_attr( $video_url ); ?>" <?php Utils::print_html_attributes( $video_params ); ?>></video>
-		<?php
+
+		if ( ! $this->render_hosted_videopress_video( $video_url, $video_params ) ) {
```

Comment:
Please add it as a video type (https://github.com/elementor/elementor/pull/20569/files#diff-075263b807b9f3432f51fac037c717bcea04e7884b5174b9d8a2f9da0ea3f79fL117)
then you can add a specific control like `videopress_url`